### DISCUSSION TITLE
Additional Spritesheet Features

### DISCUSSION THREAD

**Discussion by ghost** [Orama-Interactive/Pixelorama#448] [Ideas]
Yo, I've been using Pixelorama for Starbound sprite modding for a bit now.
It's amazing to doing edits to on that game since Pixelorama is built for such a purpose; it is far more streamlined than anything I've used.

**However, for modding purposes, there is a feature I would like to see that doesn't seem to be currently existing.**
### Feature Idea

Currently, you can import existing sprite-sheets by giving the frame parameters. 
It would be nice to be have the option to import sprite-sheets as a **new layer** in an existing project, though.
### Why?

I sometimes go use other editing programs to do things, y'see, and switching back-and-forth for specific tools can be difficult without this.

It also means that someone who was using an old sprite / image editor can import all their assets separately into Pixelorama, rather import one of the assets once then having to make a new layer and import each frame manually.

Starbound can have from 56 to 128 frames of animation at times, which makes the former, manual version a lengthy process which I will be doing shortly.
### Can it be used elsewhere?

It is useful to import multiple spritesheets that utilise the same frame dimensions; I could have the base, clothe-less sprites of a humanoid, then import the sprite sheet of existing clothes knowing that they are aligned perfectly over the humanoid and work off that for modding - this is where it would see most of the uses in terms of modding for quite a few games, I'd say.

It'd also see use in just making variants of clothing over a base sprite, really - that's where this feature would shine most too, thinking about it.

Thanks for reading.

**Comment by OverloadedOrama** [comment]
Labeling this as implemented because of #453
